WordPress.gitignore

```diff
@@ -11,3 +11,8 @@ wp-content/cache/
 wp-content/upgrade/
 wp-content/uploads/
 wp-content/wp-cache-config.php
+wp-content/plugins/hello.php
+
+/readme.html
+/license.txt
```

Comment:
Could excluding the license file from source control risk subverting the GPL?

>   For example, if you distribute copies of such a program, whether
gratis or for a fee, you must give the recipients all the rights that
you have.  You must make sure that they, too, receive or can get the
source code.  **And you must show them these terms so they know their
rights.**